tools/requirements.txt

```diff
@@ -5,192 +5,193 @@
 # requirements.txt so that is scanned by GitHub. See
```

Comment:
Sorry about the ugly diff. Poetry decided to add a whitespace before the semi-colon.